### PR TITLE
fix(vm_config): add hostSshPort and vmSshPort to Nox's vmConfig

### PR DESCRIPTION
**Pull request by folex**

### CODE REVIEW

File: cli/docs/configs/provider.md [hunk 1]
```diff
@@ -534,12 +536,14 @@ VM Network Configuration
...
-| `portRange`  | [object](#portrange) | No       | iptables-mapped port range from Host to VM                       |
-| `vmIp`       | string               | No       | Internal IP address to assign the VM                             |
+| Property      | Type                 | Required | Description                                                      |
+|---------------|----------------------|----------|------------------------------------------------------------------|
+| `publicIp`    | string               | **Yes**  | Public IP address to assign the VM. Must be publicly accessible. |
+| `bridgeName`  | string               | No       | Name of the network bridge device                                |
+| `hostSshPort` | integer              | No       | Host SSH port, default is 22                                     |
+| `portRange`   | [object](#portrange) | No       | iptables-mapped port range from Host to VM                       |
+| `vmIp`        | string               | No       | Internal IP address to assign the VM                             |
+| `vmSshPort`   | integer              | No       | VM SSH port, default is 2222                                     |
```
Comment (kmd-fl) [inline cli/docs/configs/provider.md:546]:
```suggestion
| `vmSshPort`   | integer              | No       | VM SSH port, default is 922                                     |
```
now it's like this

File: cli/docs/configs/provider.md [hunk 2]
```diff
@@ -534,12 +536,14 @@ VM Network Configuration
...
-|--------------|----------------------|----------|------------------------------------------------------------------|
-| `publicIp`   | string               | **Yes**  | Public IP address to assign the VM. Must be publicly accessible. |
-| `bridgeName` | string               | No       | Name of the network bridge device                                |
-| `portRange`  | [object](#portrange) | No       | iptables-mapped port range from Host to VM                       |
-| `vmIp`       | string               | No       | Internal IP address to assign the VM                             |
+| Property      | Type                 | Required | Description                                                      |
+|---------------|----------------------|----------|------------------------------------------------------------------|
+| `publicIp`    | string               | **Yes**  | Public IP address to assign the VM. Must be publicly accessible. |
+| `bridgeName`  | string               | No       | Name of the network bridge device                                |
+| `hostSshPort` | integer              | No       | Host SSH port, default is 22                                     |
```
Comment (kmd-fl) [inline cli/docs/configs/provider.md:543]:
```suggestion
| `hostSshPort` | integer              | No       | Host SSH port, default is 922                                     |
```

File: cli/src/lib/configs/project/provider.ts [hunk 3]
```diff
@@ -795,6 +797,16 @@ const noxConfigYAMLSchemaV1 = {
                 },
               },
             },
+            hostSshPort: {
+              nullable: true,
+              type: "integer",
+              description: `Host SSH port, default is 22`,
```
Comment (folex) [inline cli/src/lib/configs/project/provider.ts:803]:
```suggestion
              description: `Host SSH port, default is 922`,
```

File: cli/src/lib/configs/project/provider.ts [hunk 4]
```diff
@@ -795,6 +797,16 @@ const noxConfigYAMLSchemaV1 = {
...
+            },
+            vmSshPort: {
+              nullable: true,
+              type: "integer",
+              description: `VM SSH port, default is 2222`,
```
Comment (folex) [inline cli/src/lib/configs/project/provider.ts:808]:
```suggestion
              description: `VM SSH port, default is 22`,
```

**Pull request merged**